### PR TITLE
Fixes 171

### LINKED ISSUE #171: Document a risk assessment of the personal information in the system
**Is your feature request related to a problem? Please describe.**
We need a risk assessment of the information in the system so we can make the appropriate security measures.  

**Describe the solution you'd like**
Make a risk assessment and document it in the wiki based upon the information found in isuue #168 . Group 17 have worked with GDPR before and can be contacted for help to get started.

### PR DESCRIPTION
**Pull request by joandrsn**
This fixes #171 

It includes a PDF file of the potential risks that was found.

### CODE REVIEW

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw611f20 you have been assigned this. Here's your checklist.

Please use the following checklist:

**Code Documentation Only**

- [ ] It is documented which BLoCs/Screens the current class is related to
- [ ] The responsibility/functionality of the current class is explained. Also regarding the classes it is related to
- [ ] If any unusual decisions are introduced in the code, they reasoned about in the documentation
 
**Wiki Related**

- [ ] There are no dead links or references
- [ ] The markdown renders correctly in MkDocs
- [ ] The content is placed in the correct page/folder/section

You can run MkDocs locally by following [this guide](https://aau-giraf.github.io/wiki/writing_wiki/#running-locally)

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw612f20 you have been assigned this. Here's your checklist.

Please use the following checklist:

**Code Documentation Only**

- [ ] It is documented which BLoCs/Screens the current class is related to
- [ ] The responsibility/functionality of the current class is explained. Also regarding the classes it is related to
- [ ] If any unusual decisions are introduced in the code, they reasoned about in the documentation
 
**Wiki Related**

- [ ] There are no dead links or references
- [ ] The markdown renders correctly in MkDocs
- [ ] The content is placed in the correct page/folder/section

You can run MkDocs locally by following [this guide](https://aau-giraf.github.io/wiki/writing_wiki/#running-locally)

File: docs/development/security/risk_assesment.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Risk assesment
+From [issue 171](https://github.com/aau-giraf/wiki/issues/171).
+
+The risk assesment can be found here: [PDF](risk_assesment.pdf)
+
+The risk assesment have been based on the guidance of Datatilsynet.
```
Comment (antonhpalmer) [inline docs/development/security/risk_assesment.md]:
```suggestion
The risk assessment has been based on the guidance of Datatilsynet.
```

File: docs/development/security/risk_assesment.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Specifically these documents:
+
+- [Datatilsynets vejledning](https://www.datatilsynet.dk/media/7900/vejledende-tekst-om-risikovurdering.pdf)
+- [Datatilsynets artikel 25 og 32](https://www.datatilsynet.dk/media/6879/artikel25og32-vejledning.pdf)
+- [Template for risk assesment](https://sikkerdigital.dk/virksomhed/saadan-beskytter-du-din-virksomhed/skabeloner-og-vaerktoejer/)
+
+Additionally, as inspiration, ENISA has provided a guide which have also been utilised:
```
Comment (antonhpalmer) [inline docs/development/security/risk_assesment.md]:
```suggestion
Additionally, as inspiration, ENISA has provided a guide which has also been utilized:
```

**Review by antonhpalmer — CHANGES_REQUESTED** [review]
'assesment' should be changed to 'assessment' in all occurrences.

File: docs/development/security/risk_assesment.md (hunk 1 above)
Comment (DumbLol) [reply docs/development/security/risk_assesment.md]: I believe i have fixed the spelling mistakes and i also corrected the file names.

**Review by antonhpalmer — APPROVED** [review]

**Review by RasmusHRAAU — APPROVED** [review]

**Pull request merged**